### DISCUSSION TITLE
git pull issue

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1618] [Q&A] [answered]
Hi,
why I'm getting this issue while I'm trying to give git pull
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/2cf31736-83c9-4c1c-b8fe-4e640752994a)

**Comment by sreedharm07** [comment, marked as answer]
Give command **git stash** 
And then give git pull
